mmaction/models/heads/tsm_head.py

```diff
@@ -78,7 +78,8 @@ def forward(self, x, num_segments):
 
         Args:
             x (torch.Tensor): The input data.
-            num_segments (int): Number of frame segments. Default: 8.
+            num_segments (int): Useless in TSMHead.
```

Comment:
Could you provide more info, why this option exists here?

Reply:
In Recognizer2D, all [head_cls](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/recognizers/recognizer2d.py#L60) need two input params.
By default, this `num_segments == num_clips * clip_len * num_crops`, which is useful in TSN models. However, in TSM models, the `self.num_segments` we need is a required hyper-param to build models